### ISSUE TITLE
terminal only returns 'starts'

### ISSUE THREAD

**Issue by vschaefer** [fmalk/codeigniter-phpunit#31]
Hello, i despair a bit on :(
I've set up my environment, as you described it in the documentation. When I run the tests on the command line it only returns `starts` and nothing else. I've attached a screenshot of the console.
Do you have an idea where the problem could be? 
Thank you in advance for your help
![screenshot 2015-05-12 17 12 46](https://cloud.githubusercontent.com/assets/2872395/7590801/908cda54-f8ca-11e4-875c-f9ce7372c7dd.png)

Best regards
Volker Schaefer

**Comment by fmalk** [comment]
@vschaefer most likely your tests folders aren't set up properly at `phpunit.xml`.
However, I've also found errors like these before with edge PHPUnit versions, when 4.x was being introduced. But Travis runs fine with 4.5.0.

**Comment by fmalk** [comment]
@vschaefer I am currently updating the project and I revisited a similar bug, probably that's what's bugging you in some way:

CodeIgniter tries to parse command line arguments as your URI, and so any command argument can screw your test start. Since you're not using `phpunit` but `php phpunit.phar`, I imagine a `start` argument somewhere in that .phar script must be causing that.

I don't know if you're already using CI 3.0.0, but if you are, I just pushed a new project update that can help you with that, if not downright solving it. Try to debug my hack in `system/core/URI.php` to see what's been done to your `$uri` variable there.
